sql/src/main/java/io/crate/plugin/SQLPlugin.java

```diff
@@ -243,7 +248,7 @@ public Settings additionalSettings() {
             UserDefinedFunctionsMetaData::fromXContent
         ));
         entries.add(new NamedXContentRegistry.Entry(
-            IngestRulesMetaData.class,
+            MetaData.Custom.class,
```

Comment:
is this really correct here? if so why is the UserDefinedFunctionsMetaData register different and working?